cypress/integration/full_run_simple.spec.js

```diff
@@ -18,7 +18,7 @@ describe('Full run through saying "no" to everything', function() {
   // START PAGE
   it('successfully loads the home page', function() {
     cy.checkA11y()
-    cy.get('h1').should('contain', 'Claim Tax Benefits')
+    cy.get('h1').should('contain', 'File taxes to access benefits')
```

Comment:
So this this the new name of our service? @brianhendrick1 @ClementineHahn @YedidaZalik